KoalaBot.py

```diff
@@ -79,7 +79,7 @@ def get_config_from_argv():
 BOT_TOKEN = os.environ['DISCORD_TOKEN']
 BOT_OWNER = os.environ.get('BOT_OWNER')
 DB_KEY = os.environ.get('SQLITE_KEY', "2DD29CA851E7B56E4697B0E1F08507293D761A05CE4D1B628663F411A8086D99")
-COMMAND_PREFIX = "k!"
+COMMAND_PREFIX = ["k!", "K!"]

```

Comment:
I don't like this approach, it requires too many changes to `COMMAND_PREFIX[0]` than should be required.
Another Idea is to use a second constant `OPT_COMMAND_PREFIX = ["K!"]` and then merging `COMMAND_PREFIX` and `OPT_COMMAND_PREFIX` when the bot is initialised